packages/web/modals/profile.tsx

```diff
@@ -58,7 +60,6 @@ export const ProfileModal: FunctionComponent<
     accountStore,
     priceStore,
```

Comment:
Would it be possible to remove need to use price store entirely?

We have a `getAssetPrice` tRPC procedure and a use `useCoinFiatValue` hook.

Reply:
Thank you for the feedback! I have removed the price store